src/styles/blocks/calendar.scss

```
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  max-width: $maxWidth;
+  padding: 0 10 10px;
+  padding-top: 30px;
+
+  &__day {
+    @include flexBlock;
+    align-items: center;
+
+    width: $blockWidth;
+    height: $blockHeight;
+    box-sizing: border-box;
+
+    background-color: $blockColor;
+    border: 1px solid black;
+
+    font-size: 30px;
+    transition: transform 0.5s ease-out;
+
+    &:hover {
+      background-color: $hoverColor;
+      transform: translateY(-20px);
+      cursor: pointer;
+      transition: transform 0.5s ease, background-color 0.5s linear;
+    }
+
+    @for $a from 1 through 31 {
+      &:nth-child(#{$a})::before {
+        content: "#{$a}";
+      }
+    }
+  }
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $number * 101px;
```

```suggestion
      margin-left: $number * 101px;
```
would be better to put this number into variable